### ISSUE TITLE
Cannot create default configuration

### ISSUE THREAD

**Issue by grzanka** [pytrip/pytripgui#351]
I've deleted `.pytrip` folder and started fresh instance of pytripgui (installed via pip from https://github.com/pytrip/pytripgui/pull/349):

```
pip install git+https://github.com/pytrip/pytripgui.git@enchancement/292-frozen-gui-during-plan-execute
```
and removed `.pytrip` folder:
```
rm -rf ~/.pytrip
```

I've started GUI using `pytripgui` command and filled all fields in "Kernel setup" dialog:

![image](https://user-images.githubusercontent.com/7374733/104219732-c97c3c80-543e-11eb-92d3-6d33ce1da895.png)

 After clicking OK GUI crashed:

```
(pytripgui_292) grzanka@grzanka-VirtualBox:~$ pytripgui 
Traceback (most recent call last):
  File "/home/grzanka/.conda/envs/pytripgui_292/lib/python3.8/site-packages/pytripgui/app_logic/app_callbacks.py", line 80, in on_kernels_configurator
    self.app_model.settings.save()
  File "/home/grzanka/.conda/envs/pytripgui_292/lib/python3.8/site-packages/pytripgui/controller/settings_cont.py", line 87, in save
    _value = getattr(model, _attr)
AttributeError: 'MainModel' object has no attribute 'default_filename'
Aborted (core dumped)
```

![image](https://user-images.githubusercontent.com/7374733/104219778-da2cb280-543e-11eb-9c03-86dee8cc5899.png)

**Comment by grzanka** [comment]
Interesting - problem doesn't exists on master branch:
![image](https://user-images.githubusercontent.com/7374733/104220195-67700700-543f-11eb-9dcc-897ead8856ae.png)

![image](https://user-images.githubusercontent.com/7374733/104220229-7951aa00-543f-11eb-94ea-1e7c1c2a717e.png)

**Comment by grzanka** [comment]
Fixed in df0c29e
